doc/classes/TextureArray.xml

```diff
@@ -4,7 +4,20 @@
 		Array of textures stored in a single primitive.
 	</brief_description>
 	<description>
-		[TextureArray]s store an array of images in a single [Texture] primitive. Each layer of the texture array has its own mipmap chain. This makes it is a good alternative to texture atlases.
+		[TextureArray]s store an array of [Image]s in a single [Texture] primitive. Each layer of the texture array has its own mipmap chain. This makes it is a good alternative to texture atlases.
+		
```

Comment:
There shouldn't be an empty line in the description outside codeblocks. Each line == new paragraph.